_posts/2020-07-13-our-code-review-etiquette.md

```diff
@@ -0,0 +1,78 @@
+- Avoid trivialities. Think to yourself: all things considered, does it actually matter? Is the cost of the author’s time, and the potential debate, really worth it?
+- Praise the good work. You can praise the logic, design, code organization, or whatever else that you honestly felt was well done.
+- Offer alternative implementations, but assume the author already considered them: <i>"What do you think about using a custom validator here?"</i>
+- When trying to offer alternative solutions, always link to resources if possible to help ensure that the claims being raised are validated.
+- Seek to understand the author's perspective.
+- Read the commit messages if you don't understand something. It's useful to try to understand why the author decided to do something.
+- It’s OK to say it’s all good. That doesn't hurt your reputation or mean that you are a less thoughtful reviewer. If the code is okay, just sign off on the pull request with a 👍 or "Ready to merge" comment.
+
+
+#### When you have your code under review
+
+- Be grateful for the reviewer's suggestions: "Good call. I'll make that change."
+- A common axiom is <i>"Don't take it personally. The review is of the code, not you."</i> We used to include this, but now prefer to say what we mean: Be aware of how hard it is to convey emotion online and how easy it is to misinterpret feedback. If a review seems aggressive or angry or otherwise personal, consider if it is intended to be read that way and ask the person for clarification of intent in private.
+- Write good commit messages. Not just "Added x file", but "Added x class" or "Fixing x bug" if writing a summary, and if it's a complicated/weird bug that took you a while, write a long commit message with references or an explanation.
+- Keeping the previous point in mind: assume the best intention from the reviewer's comments.
+- Communication is the key. Give your reviewers context about your changes.
+- Explain why the code exists: "It's like that because of these reasons. Would it be more clear if I rename this class/file/method/variable?"
+- Extract some changes and refactorings into future tickets/stories.
+- Push commits based on earlier rounds of feedback as isolated commits to the branch. Reviewers should be able to read individual updates based on their earlier feedback.
+- Seek to understand the reviewer's perspective.
+- Try to respond to every single comment. Don’t ignore a comment you did not like.
+
+#### Enjoy!
+
+Make sure you’re enjoying what you do, and appreciate what your code reviews are achieving. There’s little point in playing the role of an intelligent computer over extended periods of time if you are unhappy about it. Get yourself your preferred slow-drinking beverage (coffee?), perhaps some snacks, a comfortable chair, and relax.
```

Comment:
@cleicar I don't understand this sentence: 

> There’s little point in playing the role of an intelligent computer over extended periods of time if you are unhappy about it.

Would it be possible to rephrase it or split it in two sentences?